plugin/admin/src/js/data/entities/Readme.md

```diff
@@ -0,0 +1,98 @@
+# Icon Data Module
+
+This module registers two icon-related entities with the WordPress Core Data store and provides custom hooks to fetch icon data. It allows you to retrieve a list of icons with filtering and pagination or a single icon by its slug.
+
+## Entities
+
+Two entities are registered:
```

Comment:
Readme states "two entities are registered" but codewise only one entity is registered.
Update accordingly.